ui/src/components/profile/ProfileDesktopView.tsx

```diff
@@ -0,0 +1,387 @@
+          New Password
+        </Typography>
+        <Divider style={{ marginTop: '2%', marginBottom: '3%' }} />
+        <TextField
+          fullWidth
+          id="outlined-adornment-password"
+          label="New Password"
+          style={{ marginBottom: '5%' }}
+          type={showNewPassword ? 'text' : 'password'}
+          {...register('newPassword')}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowNewPassword}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {showNewPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+          onChange={(e) => setNewPassword(e.target.value)}
+          onBlur={onVerifyPassword}
+        />
+        {/* TODO: error handling still aint right */}
+        <TextField
+          fullWidth
+          id="outlined-basic"
+          label="Confirm New Password"
+          style={{ marginBottom: '5%' }}
+          onChange={(e) => setConfirmPassword(e.target.value)}
+          onBlur={onVerifyPassword}
+          error={!matchPassword}
+          helperText={!matchPassword ? 'Password does not match!' : ' '}
+          type={showConfirmNewPassword ? 'text' : 'password'}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowConfirmNewPassword}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {showConfirmNewPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+        />
+        <Button variant="contained" type="submit" fullWidth>
+          <Typography variant="subtitle2">SAVE</Typography>
+        </Button>
+      </form>
+    </Stack>
+  );
+}
+
+const ProfileDesktopView: FC = (): ReactElement => {
+  const user = useSelector((state: any) => state.user);
+  const { setValue, handleSubmit, register } = useForm<UserEditorFields>();
+  const [userData, setUserData] = useState(null);
+  const [showEditProfile, setShowEditProfile] = useState<boolean>(false);
+  const [showChangePassword, setShowChangePassword] = useState<boolean>(false);
+
+  const fetchUserData = useCallback(async () => {
+    if (user.id) {
+      const { data, status } = await axios.get('/api/users/get', { params: { id: user.id } });
+      if (status === 200) {
+        setUserData(data[0]);
+        setUserInformationFields(data[0]);
+      }
+    }
+  }, [user.id]);
+
+  const handleEditUserInformation = async (data: any, e: any) => {
```

Comment:
this shod also be `UserEditorFields` no? Also if the `e` is not used, hav to delete it. Or else it can;t be deployed ltr on.

*We turned on the config to throw an error if there are any unused params.